examples/experimental/amazonlinux2023.yaml

```diff
@@ -0,0 +1,15 @@
+# This template requires Lima v0.17.0 or later and requires macOS >=13.0
+mountType: "virtiofs"
```

Comment:
this seems a bit out of place, and won't always work on qemu

if you want to change the defaults, it could in _config instead?

Reply:
This is the only supported config: https://docs.aws.amazon.com/linux/al2023/ug/kvm-supported-configurations.html

Reply:
Just wondered why it was needed here, but not in any other template

Reply:
It was needed because Amazon Linux does not provide any `sshfs` rpm